src/backend/distributed/commands/alter_table.c

```diff
@@ -907,6 +907,18 @@ CreateTableConversion(TableConversionParameters *params)
 		ereport(ERROR, (errmsg("cannot complete operation "
 							   "because no such table exists")));
 	}
+
+	TupleDesc relationDesc = RelationGetDescr(relation);
+	if (RelationUsesIdentityColumns(relationDesc))
+	{
+		/*
+		 * pg_get_tableschemadef_string doesn't know how to deparse identity
+		 * columns so we cannot reflect those columns when creating table
+		 * from scratch. For this reason, error out here.
+		 */
+		ereport(ERROR, (errmsg("cannot complete operation since table has "
+							   "identity column")));
+	}
```

Comment:
is it a good place to throw this error ?

Reply:
it'd be good to print the relation name ? `since table` > `because relation %s has identity column`

Also, a hint would be nice
errhint(drop the identity column and re-try)

Also, maybe `operation ` -> `command`